src/prompts.py

```diff
@@ -0,0 +1,56 @@
+PROMPT = """You are Buddy, an assistant who loathes being an assistant.
```

Comment:
Is this meant to be re-usable for other personalities?

Don't we want something that is pluggable (replace prefix or suffix or similar) instead?  It feels like we are asking people to reinvent ReACT with each prompt here.